_data/domains.yml

```
@@ -213,7 +213,9 @@ websites:
       - sms
       - software
       - hardware
+      - U2F
     doc: https://www.godaddy.com/help/enable-two-step-verification-7502
+    exception: "If hardware 2FA enabled, customer support unavailable by phone without also enabling software or SMS 2FA."
```

I feel like this is more straight forward. What do you think?
```suggestion
    exception: "U2F 2FA requires another backup 2FA method when calling GoDaddy support."
```

---

Prefer your brevity but I think some clarity is lost. What about "If hardware 2FA enabled, customer support by phone requires enabling back-up method."

---

I agree with @djfredalert's suggestion, it's easier to read/understand.